src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationHandler {
+    void apply(FruitTransaction transaction);
```

Comment:
OperationHandler must work with Operation
But here you work with Transaction. Think about the name. TransactionHandler, TransactionExecutor...